assets/component-facets.css

```diff
@@ -787,7 +786,6 @@ input.mobile-facets__checkbox {
 .mobile-facets__label {
   padding: 1.5rem 2rem 1.5rem 2.5rem;
   width: 100%;
-  background-color: rgb(var(--color-background));
   transition: background-color 0.2s ease;
   word-break: break-word;
   display: flex;
```

Comment:
The share is overflowing here: https://screenshot.click/01-10-te46h-wg245.png I dont see this on main (https://screenshot.click/01-10-cb4ay-29ffm.png)